src/script/script.h

```diff
@@ -187,6 +198,9 @@ enum opcodetype
     OP_NOP9 = 0xb8,
     OP_NOP10 = 0xb9,
 
+    // taproot
```

Comment:
Unify Taproot capitalization in comments.
```suggestion
    // Taproot
```

Reply:
Updated.